README.md

```diff
@@ -41,9 +41,22 @@ dnf install libtool clang-devel bison cmake ninja-build samba flex texinfo glib2
 
 # Basic usage
 
-If you want to start up a QEMU VM running CheriBSD run `cheribuild.py run-riscv64-purecap -d` (-d means build all dependencies).
-If you would like the VM to have all userspace binaries to be built as plain RISC-V binaries instead of CHERI pure-capability ones use `cheribuild.py run-riscv64-hybrid -d`.
-This will build the CHERI compiler, QEMU, CheriBSD, create a disk image and boot that in QEMU.
+The `cheribuild.py` program is normally run from within the
```

Comment:
We can always do the PATH thing as @arichardson suggests above.  I happy to always cd into the cheribuild I want because I often have several at any given moment, but that may not be typical of our downstream users.